src/components/Map.jsx

```diff
@@ -0,0 +1,116 @@
+import {useLoadScript, GoogleMap} from '@react-google-maps/api';
+import React from 'react';
+import {useState, Fragment} from 'react';
+
+import useThemeStore from '../hooks/useThemeStore';
+//import useThemeStorePersist from '../hooks/useThemeStorePersist';
+import {DayStyle} from '../mapstyles/DayStyle';
+import {NightStyle} from '../mapstyles/NightStyle';
+
+export default function Map({mapTarget, mapZoom}) {
+	// Zustand
+	const isNightMode = useThemeStore(state => state.isNightMode);
+	console.log('Map: ', isNightMode);
+
+	// The things we need to track in state
+	const [mapRef, setMapRef] = useState(null);
+	const [center, setCenter] = useState({lat: 53.53, lng: 9.99});
+
+	// Vars
+	// -Maps Background wird nicht neu gerendert
+	const backgrounds = {dayMode: '#a3c7df', nightMode: '#17263c'};
+
+	const containerStyle = {
+		width: '100%',
+		height: '100%',
+	};
+
+	React.useEffect(() => {
+		if (mapRef) {
+			//Damit kannst du pannen
+			//setCenter(mapTarget);
+			mapRef.panTo(mapTarget);
+			//alert ("lat: "+ mapTarget.lat);
+			//mapRef.moveCamera()
+			//const z = mapRef.getTilt();
+			//console.log(z);
+		}
+	}, [mapRef, mapTarget]);
+
+	// Load the Google Maps scripts
+	const {isLoaded} = useLoadScript({
+		// Enter your own Google Maps API key
+		googleMapsApiKey: 'AIzaSyCNReGHN6Uan9yZY4Fjh0DwKN43q--Tya8',
+	});
+
+	const onLoad = React.useCallback(function callback(map) {
+		//console.log("map reference: ", map)
+		setMapRef(map);
+	}, []);
+
+	React.useEffect(() => {
+		if (mapRef) {
+			//Damit kannst du pannen
+			mapRef.panTo({lat: 43.53, lng: 3.99});
+		}
+	}, [mapRef]);
+
+	function handleCenterChanged() {
+		if (mapRef) {
+			//mapRef.current.panTo({lat: 43.53, lng: 9.99});
+			const newCenter = mapRef.getCenter().toJSON();
+			setCenter(newCenter);
+			console.log(newCenter);
+			//console.log(mapRef)
+		}
+	}
+	/*
+	const loadHandler = map => {
+		// Store a reference to the Google map instance in state
+
+		//new google.maps.LatLngBounds();
+		setMapRef(map);
+		// Fit map bounds to contain all markers
+		//fitBounds(map);
+	};
+
+
+
+	 */
+
+	const renderMap = () => {
+		return (
+			<Fragment>
+				<div className="App">
+					<GoogleMap
+						options={{
+							//TODO styles je nach Zustand von switchen: 'isNightMode'
```

Comment:
I wouldn't write to-do-tasks in the code. That's what user stories are there for. And switching between languages looks inconsistent and reduces the number of people who can read your code.

Reply:
Ok. It is a feature of webstorm to organize todos in the code but i will delete them